lib/reek/configuration/app_configuration.rb

```diff
@@ -42,10 +42,9 @@ def self.from_path(path = nil)
       def self.from_map(map = {})
         allocate.tap do |instance|
           instance.instance_eval do
-            self.directory_directives = map.fetch(:directory_directives, {}).
-              extend(DirectoryDirectives)
-            self.default_directive = map.fetch(:default_directive, {}).extend(DefaultDirective)
```

Comment:
:laughing: I sense another innocent PR coming, thinly veiled in the whole unify / clean-up agenda.